src/wwwroot/controllers/DomainManagerCtrl.js

```diff
@@ -58,15 +58,17 @@
       };
 
       vm.disableDomain = function(domain) {
-        settings.createOrEditDomain(domain.name, true)
+        var onExpectedErrorResult = expectedError('domain_manager_error', loadUserDomains, 'domain_manager_error_button');
```

Comment:
What do you think about to simply add an static function `onExpectedError` in place of generate it each time?